docs/sources/developers/plugins/signing.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Signing"
+type = "docs"
++++
+
+# Signing
+
+Signing is an additional security measure to make sure plugins you authored, installed or using haven't been tampered with. Grafana will verify whether a plugin is signed or unsigned when starting up and loading plugins by inspecting and verifying its digital signature, if any.
+
+> Note: All Grafana Labs authored backend plugins, including Enterprise plugins, are signed. We're looking into providing a process for allowing  community plugins to be signed in an upcoming version of Grafana.
+
+## Signing a plugin
+
+Signing a plugin is the process of creating a signed manifest file, _MANIFEST.txt_. The first step is to create the plugin manifest. This is a JSON file that should contain the same plugin id and version found in `plugin.json`. The list of plugin files with their respective checksums (SHA256) should alow be included as `files`. Example plugin manifest:
```

Comment:
You would use grafana toolkit to create the manifest JSON, sign it and create it by calling grafana.com API.

Reply:
So idea was not to be task, rather explains how plugins are signed on a higher level.

Reply:
Hm. Then I'm not sure who's the intended audience of this page. 

If it's for end-users who are wondering what the `Signed` label stands for, then I don't think we need to explain signing to this extent, and we should put it in the general plugin documentation.

If it's for plugin developers, then we can probably leave it at explaining that it's a mechanism for Grafana-authored plugin, since they can't really do anything with the information?

Who needs to read this document, and why?

Reply:
It's purpose is to bridge the gap between the understand of and Grafana logging error "plugin is unsigned" and https://grafana.com/docs/grafana/latest/installation/configuration/#allow-loading-unsigned-plugins

Reply:
What do you suggest, that we move some parts to https://grafana.com/docs/grafana/latest/plugins/ and keep the plugin developer specific information here?

Reply:
I think that might be a good idea. Basically have a section under the general plugin category:

```
Plugins
  Overview
  Install plugins
  Signed plugins
```

that explains the what and why on a higher level. This is useful information for anyone using plugins in Grafana.

For the docs aimed at plugin developers, given that they can't sign plugins themselves, the only information that's interesting to a plugin developer at this time is: "What can/should I do in the meantime?". I don't think we need to add more details than that. 

@oddlittlebird?

Reply:
Have split up in two pages. Please have a look